var/spack/repos/builtin/packages/py-jinja2/package.py

```diff
@@ -39,3 +39,6 @@ class PyJinja2(PythonPackage):
     depends_on("py-markupsafe@0.23:", type=("build", "run"))
     depends_on("py-babel@2.7:", when="@3:+i18n", type=("build", "run"))
     depends_on("py-babel@0.8:", when="+i18n", type=("build", "run"))
+
+    # https://github.com/pallets/jinja/issues/1585
+    conflicts("^py-markupsafe@2.0.2:", when="@:2.11.3")
```

Comment:
```suggestion
    conflicts("^py-markupsafe@2.1:", when="@:2")
```
Equivalent but simpler